### ISSUE TITLE
FAQ: We need some best practices for ADC usage

### ISSUE THREAD

**Issue by ScruffR** [particle-iot/docs#735]
One permanently reoccuring source for confusion is the difference between Arduino ADCs and Particle's.
Not only the difference in resolution (12bit vs. 10bit) but also the lack of a Vref pin (Gen1 & Gen2) but even more the electronic differences regarding input impedance, lack of buffered/driven inputs, ...

Things to address might be when and why to use input/filter/decoupling caps, pull-resistors, impedance converters/buffers, ...

**Comment by rickkas7** [comment]
Good idea! Also explain how the sample time works. I'll add it to my list.
